Automattic/wp-calypso
# Add additional product data(lightbox) to calypso package

This PR adds data to be shown in lightbox to calypso-product package
#### Proposed Changes

* Adds functions in `calypso-products` to fetch **Includes**, **Benefits**, **Recommended For** for Products
* Adds the functions in `calypso-products` to fetch **Includes**, **Benefits**, **Recommended For** for Bundles
* Adds new properties to selector object

#### Testing Instructions

*Note: This is an internal change to add data to `calypso-package`, no testing in UI can be done until the components are ready. However, make sure everything works as before by following the below instructions*
* click on Jetpack Cloud live link below
    * Goto `/pricing?flags=jetpack/pricing-page-rework-v1`
* or boot up this PR 
    * Run `git fetch && git checkout add/lightbox-info-to-calypso-package`
    * Run `yarn start-jetpack-cloud`
    * Goto http://jetpack.cloud.localhost:3000/pricing?flags=jetpack/pricing-page-rework-v1
* Perform a sanity testing and confirm that everything works as before

#### Pre-merge Checklist



- [] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?



---
- Related To:
  -1202796695664022-as-1202796695664084/f

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1202796695664084
